.editorconfig

```diff
@@ -0,0 +1,16 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+[*.go]
+indent_style = space
+indent_size = 4
```

Comment:
Go uses tabs, and it is standard:
https://stackoverflow.com/questions/19094704/indentation-in-go-tabs-or-spaces

Reply:
Ofc. :man_facepalming: Sorry, fixed.